docs/aqua-book/language/crdt-maps.md

```diff
@@ -0,0 +1,57 @@
+# CRDT Maps
+
+A map is a kind of [collection](types.md#collection-types) that store arrays of elements that can be accessible by `string` keys. 
+
+```aqua
+-- A map creation
+map: %u64
+
+-- Write to a map, `foo()` returns `u64`
+map <- "key", 1234
+map <- "key", foo()
+map <- "second key", foo()
+```
+
+Access to keys and elements can create different behaviours based on tasks:
+
+```aqua
+map: %u64
+
+-- Get arrays of elements ([]u64) by key in place, trigger join-behaviour
+values <- map.get("key")
+
+-- Get stream of elements (*u64) by key, later you can join and wait for specific elements in this stream
```

Comment:
Lets devote a separate paragraph of regular text to each method? Comments are great for some not so relevant information, but for the main ideas they are not so good, because they are really hard to read. Personally, I ignore them intuitively.
![image](https://github.com/fluencelabs/docs/assets/23400549/dd91406a-2990-40b5-bdb2-10554d607ad7)